workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/WorldCommandResponseHandler.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using Improbable.Gdk.Core.Commands;
+using Unity.Entities;
+using Entity = Unity.Entities.Entity;
+
+#region Diagnostic control
+
+// ReSharper disable UnusedMember.Local
+
+#endregion
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public static partial class WorldCommandsRequirables
+    {
+        [InjectableId(InjectableType.WorldCommandResponseHandler, InjectableId.NullComponentId)]
+        [InjectionCondition(InjectionCondition.RequireNothing)]
+        public class WorldCommandResponseHandler : IInjectable
+        {
+            private readonly ILogDispatcher logDispatcher;
+
+            private readonly List<Action<WorldCommands.ReserveEntityIds.ReceivedResponse>> reserveEntityIdsDelegates;
+            private readonly List<Action<WorldCommands.CreateEntity.ReceivedResponse>> createEntityDelegates;
+            private readonly List<Action<WorldCommands.DeleteEntity.ReceivedResponse>> deleteEntityDelegates;
```

Comment:
Just noticed this is missing entity queries.

Reply:
That's another ticket

Reply:
Why is it separate?

Reply:
I don't really know. I am guessing it should be simple to add once this one is in.

Reply:
Fair enough. Easier to review with less code :P.
In that case we can probably just immediately approve it once it's up.

Reply:
Yeah sorry, that was me arbitrarily splitting the work into two jira tickets.